tests/installation/install_and_reboot.pm

```diff
@@ -26,9 +26,20 @@ sub run() {
         push(@tags, "DIALOG-packages-notifications");
         $timeout = 5500;    # upgrades are slower
     }
-    while (1) {
-        assert_screen \@tags, $timeout;
-
+    my $keep_trying = 1;
+    while ($keep_trying) {
+        # try gracefully on aarch64 because of boo#982136
+        if (get_var('ARCH') eq 'aarch64') {
```

Comment:
because it's so easy ;-) why is there a `check_var` when this comparison can be done with perl means?